packages/dd-trace/src/plugin_manager.js

```diff
@@ -161,10 +166,6 @@ module.exports = class PluginManager {
       sharedConfig.clientIpEnabled = clientIpEnabled
     }
 
-    sharedConfig.site = site
-    sharedConfig.url = url
-    sharedConfig.headers = headerTags || []
-
```

Comment:
In addition to adding `ciVisibilitySessionName`, I refactored of the `sharedConfig.*` assignments, which could be done at once